### PR TITLE
Kissplice

### PR DESCRIPTION
**Pull request by cmonjeau**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/kissplice/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+requirements:
+  build:
+    - gcc   # [linux]
+    - llvm  # [osx]
+    - cmake
+    - zlib
+  run:
+    - libgcc  # [linux]
+    - zlib
```
Comment (bgruening) [inline recipes/kissplice/meta.yaml]: can you pin zlib against CONDA_ZLIB here.

Comment (cmonjeau) [reply recipes/kissplice/meta.yaml]: Done

File: recipes/kissplice/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc   # [linux]
+    - llvm  # [osx]
+    - cmake
+    - zlib {{CONDA_ZLIB}}*
+    - python 2.7.9
```
Comment (bgruening) [inline recipes/kissplice/meta.yaml]: please don't pin python here, skip it in the build section with `skip: True  # [py3k]`

File: recipes/kissplice/build.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+mkdir -p build
+cd build
+
+# cmake command
+cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} ..
+
+# make commands
+make -j1
+#make test --debug
```
Comment (bgruening) [inline recipes/kissplice/build.sh]: why is this skipped?

File: recipes/kissplice/meta.yaml (hunk 2 above)
Comment (cmonjeau) [reply recipes/kissplice/meta.yaml]: done

File: recipes/kissplice/build.sh (hunk 3 above)
Comment (cmonjeau) [reply recipes/kissplice/build.sh]: Oops. Changed!

**Comment by bgruening** [comment]
Great work, thanks @cmonjeau!

**Pull request merged**